content-src/components/ASRouterAdmin/ASRouterAdmin.scss

```diff
@@ -209,7 +209,7 @@
     }
   }
 
-  .dsEnabled {
+  .dsCheckbox {
```

Comment:
is this used?

Reply:
Not anymore, good catch.

(I was initially using it but renaming it until I decided on a different approach, but this change got left in)